README.md

```diff
@@ -1,164 +1,54 @@
-juju add-unit mongodb -n <num_of_replicas_to_add>
-```
-The implementation of `add-unit` allows the operator to add more than one unit, but functions internally by adding one replica at a time, as specified by the [constraints](https://www.mongodb.com/docs/manual/reference/command/replSetReconfig/#reconfiguration-can-add-or-remove-no-more-than-one-voting-member-at-a-time) of MongoDB.
+Charmed MongoDB (VM Operator) is an enhanced, open source and fully-compatible drop-in replacement for the MongoDB Community Edition with advanced MongoDB enterprise features. This operator uses the [Charmed MongoDB snap package](https://snapcraft.io/charmed-mongodb), which offers more features than the MongoDB Community version, such as backup and restores, monitoring and security features.
 
+To see the Charmed MongoDB features and releases, visit our [Release Notes page](https://github.com/canonical/mongodb-operator/releases). Currently the charm supports:
+- Replication
+- Password Rotation
+- User management
+- TLS
+- Backup & Restore
 
-#### Removing Replicas 
-Similarly to scale down the number of replicas the `juju remove-unit` functionality may be used i.e.
-```shell
-juju remove-unit <name_of_unit1> <name_of_unit2>
-```
-The implementation of `remove-unit` allows the operator to remove more than one replica so long has the operator **does not remove a majority of the replicas**. The functionality of `remove-unit` functions by removing one replica at a time, as specified by the [constraints](https://www.mongodb.com/docs/manual/reference/command/replSetReconfig/#reconfiguration-can-add-or-remove-no-more-than-one-voting-member-at-a-time) of MongoDB.
+## Charm version, environment and OS
 
+A charm version is a combination of both the application version and / (slash) the channel, e.g. 5/stable, 5/candidate, 5/edge. The channels are ordered from the most stable to the least stable, candidate, and edge. More risky channels like edge are always implicitly available. So, if the candidate is listed, you can pull the candidate and edge. When stable is listed, all three are available. 
 
-## Relations
+You can deploy the charm a stand-alone machine or cloud and cloud-like environments, including AWS, Azure, OpenStack and VMWare.
 
-Supported [relations](https://juju.is/docs/olm/relations):
+The upper portion of this page describes the Operating System (OS) where the charm can run e.g. 5/stable is compatible and should run in a machine with Ubuntu 22.04 OS.
 
-#### New `mongodb_client` interface:
 
-Relations to new applications are supported via the `mongodb_client` interface. To create a relation: 
+## Security, Bugs and feature request
 
-```shell
-juju relate mongodb application
-```
+If you find a bug in this snap or want to request a specific feature, here are the useful links:
 
-To remove a relation:
-```shell
-juju remove-relation mongodb application
-```
+* Raise issues or feature requests in [Github](https://github.com/canonical/mongodb-operator/issues)
 
```

Comment:
Meet the community and chat with us if there are issues and feature requests in our [Mattermost Channel](https://chat.charmhub.io/charmhub/channels/data-platform).